tests/cartesian_tests/unit_tests/test_exec_info.py

```diff
@@ -69,9 +70,18 @@ def diffusion_def(
                 flux_x[0, 0, 0] - flux_x[-1, 0, 0] + flux_y[0, 0, 0] - flux_y[0, -1, 0]
             )
 
-    def compile_stencils(self, backend):
-        self.advection = gtscript.stencil(backend=backend, definition=self.advection_def)
-        self.diffusion = gtscript.stencil(backend=backend, definition=self.diffusion_def)
+    def compile_stencils(self, backend, worker_id):
```

Comment:
We could document here , that `worker_id` comes from pytest-xdist but it's not critical

Reply:
Alternatively, there is also the option to not base `name` on `worker_id` but instead just use different names for test_backcompatibility and test_accumulate.